app/book-detail/detail.template.html

```diff
@@ -0,0 +1,69 @@
+<a ui-sref="books" class="button-back"> < Volver</a>
+<div class="container-detail">
+  <div class="container-info">
+    <div class="image-detail">
+      <img ng-class="$ctrl.detail.image_url  ? 'image' : 'default'" ng-src="{{$ctrl.detail.image_url ? $ctrl.detail.image_url : $ctrl.imageUrl}}"/>
```

Comment:
Add alt attr please. And adding the controller name to the route file you could avoid the $ctrl.